mistralrs-pyo3/src/lib.rs

```diff
@@ -122,6 +124,7 @@ impl Runner {
                 response: tx,
                 return_logprobs: request.logprobs,
                 is_streaming: request.stream,
+                constraint: Constraint::None,
```

Comment:
I never dealt with Py/Rust FFI. Python API has no support for constraints...

Reply:
I will add this is a seperate PR.